src/perception/vision_pipeline/conda_requirements

```diff
@@ -0,0 +1,11 @@
+# REMEMBER to update deps in jetson_install.sh as well
+opencv==4.5.2
+numpy==1.21.4
+matplotlib==3.5.1
+requests==2.26.0
+tqdm>=4.41.0
+pandas>=1.1.4
+seaborn>=0.11.0
+pytorch
```

Comment:
put `pytorch` and `torchvision` above the reminder comment above (or make a note of it in the comment or something), so its obvious they dont need to be in the `jetson_requirements.txt` file.

Reply:
probably `ros-galactic-cv-bridge` too